airflow/models/baseoperator.py

```diff
@@ -1048,6 +1061,14 @@ class BaseOperatorLink(metaclass=ABCMeta):
     Abstract base class that defines how we get an operator link.
     """
 
+    operators = None   # type: Optional[List[BaseOperator]]
```

Comment:
```suggestion
    operators = []   # type: List[Type[BaseOperator]]
```

Oh also we want a list of classes, not a list of objects!

Reply:
Updated